building-a-preview/index.md

```diff
@@ -0,0 +1,777 @@
+  status in yellow. This indicates a complete Preview rebuild from the beginning
+  of the build process, so it should take as long as a typical build.
+- **Refreshing:** When a refresh has been kicked off, you'll see a `refreshing`
+  status in yellow. This indicates a Preview that is pulling in the latest code
+  from git, and then running any commands in the `update` section, followed by
+  the `build` section of the
+  [Configuration file](../setting-up-services/index.md#create-a-tugboat-config-file).
+- **Resuming:** When you've used the Action -> Start option, you'll see a
+  `resuming` status in yellow while the Preview starts spinning up
+  [services](../setting-up-services/index.md) again.
+- **Stopping:** When you've used the Action -> Stop option, you'll see a
+  `stopping` status in yellow while the Preview goes through the process of
+  stopping [services](../setting-up-services/index.md).
+- **Stopped:** When you've used the Action -> Stop option, you'll see a
+  `stopped` status in red to indicate that the Preview has successfully stopped
+  [services](../setting-up-services/index.md).
+- **Suspended** - When Previews have been inactive for a period of time, you'll
+  see a `suspended` status. Any incoming HTTP request to the preview will
+  automatically start it again.
+- **Failed:** When something goes wrong during the the last action that was
+  taken on the Preview, you'll see a `failed` status in red. Details should be
+  available in the Preview's activity logs. Sometimes a failed Preview can be
+  recovered by resetting it. For more help with a `failed` preview, take a look
+  at our [troubleshooting](../setting-up-services/index.md) docs, or go to our
+  [Help and Support](../help-and-support/index.md) page to join our Slack
+  support channel or email us.
+- **Unavailable** - When something goes wrong trying to load the Preview, you
+  may see an `unavailable` status. This usually indicates an internal Tugboat
+  error. [Resetting](#reset) a Preview often fixes this.
+- **Canceled** - When you cancel a Preview while it's building, you'll see a
+  `canceled` status in red.
+
+### Service states
+
+A Service could be in any of the above states, as well as:
+
+- **Committing** - Tugboat is currently taking a snapshot of the current state
+  of the Service.
+- **Waiting** - Tugboat is performing some operation on the Service's repo, and
+  the Service is waiting for its turn.
+
+## Set a Base Preview
+
+If you want to speed up your Preview builds, and make subsequent Preview builds
+smaller files, you can set a Base Preview as a starting point for subsequent
+Previews.
+
+- [How Base Previews work](#how-base-previews-work)
+- [How to set a Base Preview](#how-to-set-a-base-preview)
+- [Using multiple Base Previews](#using-multiple-base-previews)
+- [Building a Preview from scratch after you've set a Base Preview](#building-a-preview-from-scratch-after-youve-set-a-base-preview)
+
+### How Base Previews work
```

Comment:
I think this information needs to be presented higher up. We talk a lot about using base previews without ever explaining what they are or why you would want to use them.

Reply:
Noted - I'll think about this as I ponder reorganizing this page. I wanted to give users the basic entry point into using Tugboat; i.e. building your first Preview, and then sharing it - and then introduce them to the concept of Base Previews. I suppose I could move the info about the build process and the Preview Actions, and move the Base Preview stuff up... I may just need to print this stuff out and look at it on paper.

Anywho, noted, and I'm open to conversation about a good order to present this info if you have strong opinions one way or another!